HelpSource/Tutorials/JITLib/jitlib_asCompileString.schelp

```diff
@@ -47,3 +47,43 @@ p.document;		// document everything
 p.document([\out]); 	// document all dependants of ~out
 p.document([\ctl1]);	// document all dependants of ~ctl1
 ::
+
+Ndefs and NodeProxies can also store their full state as a code string:
+code::
+// Ndef with source
+Ndef(\x, { Saw.ar(\freq.kr(234), 0.1) });
+Ndef(\x).asCode;
+// returns:
+"Ndef('x', { Saw.ar(\freq.kr(234), 0.1) });
+"
+
+// Ndef with source and settings
+Ndef(\a,  { Saw.ar(\freq.kr, 0.1) }).set(\freq, 123);
+Ndef(\a).asCode;
+// returns:
+"(
+Ndef('a', { Saw.ar(\freq.kr, 0.1) });
+Ndef('a').set('freq', 123);
+);
+"
+
+// anonymous nodeproxy - returns example code with globar var a = ...
```

Comment:
global

Reply:
Maybe should be interpreter var to be finicky.

Reply:
fixed...